terraform/backend/module/alb.tf

```diff
@@ -1,12 +1,41 @@
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
+
+resource "aws_s3_bucket" "log-bucket" {
+  bucket_prefix = "${var.env}-${var.app}-external-alb-logs"
+}
+
+data "template_file" "log-bucket-policy" {
+  template = "${file("${path.module}/log-bucket-policy.json.tpl")}"
+
+  vars = {
+    alb_account_id = "${lookup(var.aws_region_map, "${data.aws_region.current.name}")}"
+    s3_arn         = "${aws_s3_bucket.log-bucket.arn}"
+    account_id     = "${data.aws_caller_identity.current.account_id}"
+  }
+
+resource "aws_s3_bucket_policy" "attach-policy-to-log-bucket" {
+  bucket = "${aws_s3_bucket.log-bucket.id}"
+  policy = "${data.template_file.log-bucket-policy.rendered}"
+}
+
+
 # alb
 resource "aws_lb" "backend" {
+  depends_on = [
```

Comment:
not needed since you declare the resource in the block below

Reply:
Truth, the dependency declared for logical reasons, but removing it should not create any impact. 
I'll follow your request

Reply:
Addressed in https://github.com/codersagainstcovidorg/infra/pull/5/commits/da8ac92be8dd83818b41b3de9515f92839d6e27b